web/docs/guides/auth/auth-facebook.mdx

```diff
@@ -0,0 +1,116 @@
+---
+id: auth-facebook
+title: "Facebook Login"
+description: Setting up Facebook Login
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Setting up Facebook Login
+
+## Description
+
+Adding a Facebook Login button to your application is easy if you follow these steps.
+
+The setup process consists of three parts:
+
+* Create and configure a Facebook Application on the [Facebook Devalopers Site](https://developers.facebook.com)
+* Add your Facebook keys to your [Supabase Project](https://supabase.io)
+* Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js)
+
+## Environment
+
+* [Supabase JS Client](https://github.com/supabase/supabase-js)
+* [Supabase Dashboard (UI)](https://supabase.io)
+* [Facebook Developers Dashboard](https://developers.facebook.com/)
+
+## Requirements
+* Supabase account (Free Tier OK)
+* Any existing Supabase Project
+* Javascript-based client application (Angular, React, Vue, etc.)
+
+## Steps
+
+### Create a Facebook Developer Account
+
+First, you'll need a Developer Account at [https://developers.facebook.com](https://developers.facebook.com).
+
+### Create a Facebook App
+
+Once logged into *"Facebook for Developers"*:
+
+- Click on "My Apps" at the top right.
+- Click "Create App" near the top right.
+- Select your app type and click "Continue".
+- Fill in your app information, then click "Create App".
+- This should bring you to the screen: "Add Products to Your App". (Alternatively you can click on "Add Product" in the left sidebar to get to this screen.)
+
+### Set up FaceBook Login for your Facebook App
+
+From the "Add Products to your App" screen:
+
+- Click "Setup" under "Facebook Login"
+- Skip the Quickstart screen, instead, in the left sidebar, click "Settings" under "Facebook Login"
+- Enter your callback URI under "Valid OAuth Redirect URIs" on the "Facebook Login Settings" page
+    - Your OAuth Redirect URI looks like this:
+      - https://xxxxxxxxxxxxxxxx.supabase.co/auth/v1/callback
```

Comment:
let's wrap these in code ticks - `https://project-ref.supabase.co/auth/v1/callback`